### PR TITLE
#153741758 landing page Users should see a landing page with effective UI/UX

### PR DESCRIPTION
**Pull request by SEUNAGBEYE**
### What does this PR do?

- This PR allows a user to see the landing page on visiting the site
- Helps a user get a brief info about the game

### Description of Task to be completed?

N/A

### How should this be manually tested?

- Run npm start to start the server
- Go to localhost:3000 to see the landing page

screen shot 2018-01-08 at 4 33 51 pm

### Any background context you want to provide?

N/A

### What are the relevant pivotal tracker stories?

- Users should see a landing page with effective UI/UX #153741758

### 
Questions:

N/A

### CODE REVIEW

File: app/controllers/index.js [hunk 1]
```diff
@@ -1,23 +1,23 @@
 /**
  * Module dependencies.
  */
-var mongoose = require('mongoose'),
-    async = require('async'),
-    _ = require('underscore');
+let mongoose = require('mongoose'),
```
Comment (chinezenwosu) [inline app/controllers/index.js]: Good catch.

File: app/controllers/index.js [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
+  async = require('async'),
```
Comment (chinezenwosu) [inline app/controllers/index.js]: Prefer one const/let declaration per variable.

Comment (chinezenwosu) [inline app/controllers/index.js]: Prefer `const` over `let` when the variable will never be changed.

File: app/controllers/index.js [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
+  _ = require('underscore');
 
 /**
  * Redirect users to /#!/app (forcing Angular to reload the page)
  */
-exports.play = function(req, res) {
+exports.play = function (req, res) {
```
Comment (chinezenwosu) [inline app/controllers/index.js]: Use es6 `export` over `exports`. Change all instances of this.

File: config/routes.js [hunk 4]
```diff
@@ -1,93 +1,92 @@
...
+
+  app.get('/auth/google/callback', passport.authenticate('google', {
+    failureRedirect: '/signin'
+  }), users.authCallback);
+
+  // Finish with setting up the userId param
+  app.param('userId', users.user);
+
+  // Answer Routes
+  const answers = require('../app/controllers/answers');
```
Comment (chinezenwosu) [inline config/routes.js]: Put all require or import statements at the top of the file

**Comment by kensanni** [comment]
Great and nice-looking UI, It would look more better if you could change the font to something  more playful

**Pull request merged**